docs/providers/mongodb.md

```diff
@@ -0,0 +1,33 @@
+# MongoDB
+
+Featureform supports [MongoDB](https://www.mongodb.com/) as an Inference Store.
+
+## Implementation
+
+A MongoDB table is created for every feature. A metadata table exists within the Database as well to allow the provider to keep track of its own state. Featureform's scheduler aims to achieve consistency between MongoDBs's internal state with the user's desired state as specified in the metadata service.
+
+## Configuration
+
+First we have to add a declarative MongoDB configuration in Python. In the following example, only name is required, but the other parameters are available.
```

Comment:
Wouldn't host and port also be required? Credentials? Are there combinations of valid providers?

Reply:
You right

Reply:
Fixed